UACloudLibraryGraphQLClient/UACloudLibClientLibrary/Models/QueryModel.cs

```diff
@@ -70,7 +72,28 @@ public class VariableResult
         public string Browsename { get; set; }
         [JsonProperty("variabletype_value")]
         public string Value { get; set; }
-        [JsonProperty("vaiabletype_namespace")]
+        [JsonProperty("variabletype_namespace")]
         public string Namespace { get; set; }
     }
+
+    public class BasicNodesetInformation
+    {
+        [JsonProperty("nodeset_id")]
+        public uint ID { get; set; }
+
+        [JsonProperty("nodesettitle")]
+        public string Title { get; set; }
+
+        [JsonProperty("orgname")]
+        public string Organisation { get; set; }
+
+        [JsonProperty("license")]
+        public AddressSpaceLicense License { get; set; }
+
+        [JsonProperty("addressspacecreationtime")]
```

Comment:
Are we going with PublicationTime here? If so, this and UANodesetResult should both be changed.

Reply:
As far as I'm aware, it's the nodesets creation time